code/game/gamemodes/burglars/burglars.dm

```diff
@@ -4,7 +4,7 @@
 	name = "burglars"
 	config_tag = "burglars"
 	required_players = 5
-	max_players = 15
+	max_players = 30
```

Comment:
I dont actually think this gamemode works with 2 antags at 30 players. Maybe 20 max?